packages/hypertest-core/src/init.ts

```diff
@@ -0,0 +1,107 @@
+import path from 'node:path';
+import fs from 'node:fs/promises';
+import inquirer from 'inquirer';
+
+const QUESTIONS = [
+  {
+    type: 'number',
+    name: 'concurrency',
+    message: 'Set concurrency:',
+    default: 30,
+  },
+  {
+    type: 'input',
+    name: 'imageName',
+    message: 'Set image name:',
+  },
+  {
+    type: 'input',
+    name: 'localImageName',
+    message: 'Set local image name:',
+  },
+  {
+    type: 'input',
+    name: 'localBaseImageName',
+    message: 'Set local base image name:',
+  },
+  {
+    type: 'list',
+    name: 'testRunnerOption',
+    message: 'What test runner you want to use:',
+    choices: ['playwright'],
+  },
+  {
+    type: 'input',
+    name: 'AWScloudFunctionProvider_baseImage',
+    message: 'Cloud function base image:',
+  },
+  {
+    type: 'input',
+    name: 'AWScloudFunctionProvider_region',
+    message: 'Cloud function region:',
+    default: 'eu-central-1',
+  },
+  {
+    type: 'input',
+    name: 'AWScloudFunctionProvider_ecrRegistry',
+    message: 'Cloud ECR registry URL:',
+    default: 'dkr.ecr.eu-central-1.amazonaws.com',
+  },
+  {
+    type: 'input',
+    name: 'AWScloudFunctionProvider_functionName',
+    message: 'Cloud function name:',
+  },
+  {
+    type: 'input',
+    name: 'AWScloudFunctionProvider_bucketName',
+    message: 'Cloud storage bucket name:',
+  },
+];
+
+export const initializeHypertestConfig = async () => {
+  const configPath = path.resolve(process.cwd(), 'hypertestTEST.config.js');
```

Comment:
I think we need to change `hypertestTEST.config.js` to proper file name and it will be good to move it as some variable (at least top file level)?